### PR TITLE
Code formating

### PR DESCRIPTION
**Pull request by SmetDenis**
Hi there,

I've figured out that you have issues with code formating in your Readme file.

Feel free to add your comments, please.

Have a nice day :)

### CODE REVIEW

**Comment by julijaschwarz** [comment]
Thank you! Look's good for now.
But for some reason you did some additional changes with badge. Can you please revert this line.

File: README.md [hunk 1]
```diff
@@ -45,4 +47,4 @@ Type the name of any game and press Enter.
 
 [![asciicast](https://asciinema.org/a/b1cclrAaNRQgblwsl9lMcNgDJ.svg)](https://asciinema.org/a/b1cclrAaNRQgblwsl9lMcNgDJ)
 
-[![asciicast](https://asciinema.org/a/ZOez5X2uEzuWdcTGyuLNl2M7v.svg)](https://asciinema.org/a/ZOez5X2uEzuWdcTGyuLNl2M7v)
\ No newline at end of file
+[![asciicast](https://asciinema.org/a/ZOez5X2uEzuWdcTGyuLNl2M7v.svg)](https://asciinema.org/a/ZOez5X2uEzuWdcTGyuLNl2M7v)
```
Comment (julijaschwarz) [inline README.md:50]: This one.

**Review by julijaschwarz — CHANGES_REQUESTED** [review]

**Comment by SmetDenis** [comment]
Oh, thanks. Nice catch!

File: README.md (hunk 1 above)
Comment (SmetDenis) [reply README.md:50]: No prob

**Pull request merged**